From: Gunnar Wrobel <wrobel@pardus.de>
Subject: [PATCH] Kolab_Format_1.0.1_p1.patch

Removes an additional option that does not help in dealing with XML format
problems but instead breaks encoding again.

https://www.intevation.de/roundup/kolab/issue3528

STATUS: MERGED

http://cvs.horde.org/diff.php/framework/Kolab_Format/lib/Horde/Kolab/Format/XML.php?r1=1.15&r2=1.16

Signed-off-by: Gunnar Wrobel <wrobel@pardus.de>

Index: a/framework/Kolab_Format/lib/Horde/Kolab/Format/XML.php
===================================================================
RCS file: /repository/framework/Kolab_Format/lib/Horde/Kolab/Format/XML.php,v
retrieving revision 1.5.2.11
retrieving revision 1.5.2.12
diff -u -B -r1.5.2.11 -r1.5.2.12
--- a/framework/Kolab_Format/lib/Horde/Kolab/Format/XML.php      2 Apr 2009 20:14:51 -0000       1.5.2.11
+++ a/framework/Kolab_Format/lib/Horde/Kolab/Format/XML.php      8 Apr 2009 18:34:39 -0000       1.5.2.12
@@ -597,7 +597,7 @@
     {
         $params = array(
             'xml' => $xmltext,
-            'options' => HORDE_DOM_LOAD_REMOVE_BLANKS | HORDE_DOM_LOAD_RECOVER,
+            'options' => HORDE_DOM_LOAD_REMOVE_BLANKS,
         );
 
         $result = Horde_DOM_Document::factory($params);
